snmp/tests/compose/data/riverbed-interceptor.snmprec

```diff
@@ -0,0 +1,272 @@
+1.3.6.1.2.1.1.1.0|4|riverbed-interceptor Device Description
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.17163.1.3
+1.3.6.1.2.1.1.5.0|4|riverbed-interceptor.device.name
+1.3.6.1.2.1.2.1.0|2|28
+1.3.6.1.2.1.25.1.1.0|67|4163926289
+1.3.6.1.2.1.25.1.2.0|4x|07d209150d352003
+1.3.6.1.2.1.25.1.3.0|2|13
+1.3.6.1.2.1.25.1.4.0|4x|74686569722061637465642061637465642064726976696e67
+1.3.6.1.2.1.25.1.5.0|66|2794395881
+1.3.6.1.2.1.25.1.6.0|66|211929330
+1.3.6.1.2.1.25.1.7.0|2|10
+1.3.6.1.2.1.25.2.2.0|2|10
+1.3.6.1.2.1.25.2.3.1.1.8|2|8
+1.3.6.1.2.1.25.2.3.1.1.15|2|15
+1.3.6.1.2.1.25.2.3.1.2.8|6|1.3.6.1.3.90.198.199.50.64.115
+1.3.6.1.2.1.25.2.3.1.2.15|6|1.3.6.1.3.90.9.38.139.10.24.199.127
+1.3.6.1.2.1.25.2.3.1.3.8|4x|6f78656e207a6f6d626965732062757420717561696e746c7920666f7277617264206b65707420717561696e746c79206f78656e
+1.3.6.1.2.1.25.2.3.1.3.15|4x|6275742062757420666f7277617264207468656972204a61646564204a6164656420717561696e746c79206f78656e20717561696e746c79
+1.3.6.1.2.1.25.2.3.1.4.8|2|13
+1.3.6.1.2.1.25.2.3.1.4.15|2|25
+1.3.6.1.2.1.25.2.3.1.5.1|66|285
```

Comment:
Is it okay that I gave it my own value? Does it matter?

Reply:
The values don't matter as long as they are of the correct type and don't have sensitive information